modules/custom/d_color/js/color.js

```diff
@@ -0,0 +1,38 @@
+(function ($, Drupal) {
+
+  'use strict';
+
+  Drupal.behaviors.dpColorpicker = {
+    attach: function (context, settings) {
+      $(context).ready(function() {
+        $('.form-type-d-color', context).once().each(function () {
+          var $element = $(this);
+
+          if ($element.hasClass('binded')) {
+            return;
+          }
+
+          $element.addClass('binded');
```

Comment:
a to jest uzywane tylko do tego, zeby nie przetwarzac elementu "kolejny" raz? czy to nie bedzie załatwione przez  `once`   ?